### DISCUSSION TITLE
Android ADB instructions

### DISCUSSION THREAD

**Discussion by munaaf** [PRProd/HA-Firemote#495] [General]
The video linked in step 1 fails to cover a configuration needed to setup the android bridge integration.

Under usb debugging, you need to enabled network debugging (at least on shield pro 2019). This caused confusion and failure to enable bridge integration until I turned on.

Could we add to the readme guide?

**Comment by PRProd** [comment]
Greetings @munaaf!

Thank you for letting me know about that!  I sincerely apologize for the missing instruction.  [I've added it to the written instructions](https://github.com/PRProd/HA-Firemote/blob/main/README.md#download-and-setup-via-home-assistant-ui--hacs) for now, and will do my best to remember to include it when I update that video in the future.

Do you think that the updated written instructions make enough sense to help someone else that was trying to get their NVIDIA Shield connected through ADB?

-Doug

**Reply by munaaf** [reply]
Ty! Clear to me, and found a few posts on HA forums without a solution of people stuck on bridge setup too, this helps for now!
